packages/turf-isobands/test.js

```diff
@@ -0,0 +1,17 @@
+var test = require('tape');
+var fs = require('fs');
+var isobands = require('./');
+
+test('isobands', function(t){
+  var points = JSON.parse(fs.readFileSync(__dirname + '/geojson/Points.geojson'));
```

Comment:
Try to make your tests have dynamic data loading instead of hard coded fixtures in your tests.
That way someone can expand easily the test cases by simply adding a new `.geojson` to the `test/in`.

`@turf/union` has a good example of that:
https://github.com/Turfjs/turf/blob/master/packages/turf-union/test.js